galaxy.yml

```
@@ -7,7 +7,8 @@ name: qradar
 # a list of the collection's content authors: 'Full Name <email> (http://site) @nicks:irc/im/site#channel'
 authors:
   - Ansible Security Team (https://github.com/ansible-security)
-
+dependencies:
+  "ansible.netcommon": ">=2.5.0"
```

This has to be 2.5.1 to work
```suggestion
  "ansible.netcommon": ">=2.5.1"
```